src/BookNavigator/volumes/volumes-provider.js

```diff
@@ -24,40 +24,52 @@ export default class VolumesProvider {
     this.id = 'volumes';
     this.label = `Viewable files (${this.volumeCount})`;
     this.icon = html`${volumesIcon}`;
-    this.actionButton = this.headerIcon;
-    this.sortVolumes(true);
+
+    this.sortOrderBy = "";
+    this.sortVolumes("initial");
+  }
+
+  get sortInitialIcon() {
+    return html`<button class="sort-by neutral-icon" aria-label="Sort volumes in initial order" @click=${() => this.sortVolumes('asc')}>${sortNeutralIcon}</button>`;
   }
 
   get sortAscendingIcon() {
-    return html`<button class="sort-asc icon" aria-label="Sort volumes in ascending order" @click=${() => this.sortVolumes()}>${sortAscendingIcon}</button>`;
+    return html`<button class="sort-by asc-icon" aria-label="Sort volumes in ascending order" @click=${() => this.sortVolumes('desc')}>${sortAscendingIcon}</button>`;
   }
 
   get sortDescendingIcon() {
-    return html`<button class="sort-desc icon" aria-label="Sort volumes in descending order" @click=${() => this.sortVolumes()}>${sortDescendingIcon}</button>`;
+    return html`<button class="sort-by desc-icon" aria-label="Sort volumes in descending order" @click=${() => this.sortVolumes('initial')}>${sortDescendingIcon}</button>`;
   }
 
   get headerIcon() {
-    return this.isSortAscending ? this.sortAscendingIcon : this.sortDescendingIcon;
+    if (this.sortOrderBy === 'initial') return this.sortInitialIcon;
+    else if (this.sortOrderBy === 'asc') return this.sortAscendingIcon;
+    else return this.sortDescendingIcon;
```

Comment:
This is getting bigger now, so we might want to convert to an object:

```suggestion
    // Put this in the constructor, perhaps?
    const sortIcons = {
        index: html`<button ... >`,
        title_asc: html`<button ...>`,
        title_desc: html`<button ...>`,
    };
    
    return sortIcons[this.sortOrderBy];
```

This will let us remove data from the getter names ( <code>sort<b>Initial</b>Icon</code> ).